# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/webauthn/WebAuthnUtil.h
# Commit: 287bedb227e8
# Full Hash: 287bedb227e89933347633242efe1acd4236ad70
# Author: John Schanck <jschanck@mozilla.com>
# Date: 2025-02-12 09:32:07
# Regressor Bug: 1945969
# File Overlap Count: 1
# Description:
#   Bug 1945969 - part 5: rename WebAuthnManager to WebAuthnHandler. r=keeler
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D237257
# ==============================================================================

diff -r a01c165a2524 -r 287bedb227e8 dom/webauthn/WebAuthnUtil.h
--- a/dom/webauthn/WebAuthnUtil.h	Wed Feb 12 02:12:58 2025 +0000
+++ b/dom/webauthn/WebAuthnUtil.h	Wed Feb 12 02:12:58 2025 +0000
@@ -7,10 +7,6 @@
 #ifndef mozilla_dom_WebAuthnUtil_h
 #define mozilla_dom_WebAuthnUtil_h
 
-/*
- * Utility functions used by both WebAuthnManager and U2FTokenManager.
- */
-
 #include "mozilla/dom/WebAuthenticationBinding.h"
 #include "ipc/IPCMessageUtils.h"
 